### DISCUSSION TITLE
Add the ability to see warn/ban/mute history of players

### DISCUSSION THREAD

**Discussion by DrakePork** [Zrips/CMI#2169] [Feature Requests]
**Description of feature/improvement request:**
Improve upon the current ban system CMI offers by adding the ability to see how many times a player has been banned/muted/warned & the reasoning behind the punishments.

---
### Detailed explanation of what it should do, when and in what situation: ###
Basically if you for example type the commands /history (player), /warnhistory (player), /banhistory (player) or /mutehistory (player) you get up a list of bans/warnings a player has recieved & the reasons they were warned/banned/muted. /history would show all of them while the other commands would show the specific ones.

### Basic explanation as to why this feature is needed and/or useful to have: ###
This feature is useful to have when you're checking if a player has been banned before, how many times they've been banned/muted/warned etc, which could be useful if you're appointing a staff member.

**Comment by Jaeger-Tim** [comment]
I was about to make the same request when I saw this one.
